runtests.py

```diff
@@ -2118,6 +2132,9 @@ def main():
     if options.xml_output_dir:
         shutil.rmtree(options.xml_output_dir, ignore_errors=True)
 
+    interval = 10
```

Comment:
Style nit: it'd be cleaner to not re-assign (and perhaps name this something less generic).

Reply:
fixed